### DISCUSSION TITLE
debugger shuts down

### DISCUSSION THREAD

**Discussion by raimoncasamitjana** [TheAlbertDev/MASB#52] [Q&A]
When trying to debug the code from last session (7) the debugger won't connect and shuts down automatically. I tried to reboot both the project and computer but it keeps appearing. This problem happens to me when implementing the challenge, but it seems to work fine without it. 

![image](https://user-images.githubusercontent.com/78802512/114932415-c62a8f80-9e37-11eb-820d-af650ca7c13f.png)

**Comment by Albert-Alvarez** [comment]
It is difficult to remotely troubleshoot problems specific to your computers or boards. We'd better look at it during the lab sessions.

Even so, try what is discussed in this thread on the official STM forum. It seems that the user had the same problem (a STM32CubeIDE bug) and has managed to fix it.

https://community.st.com/s/question/0D50X0000BxzLw0/how-can-i-remove-error-like-waiting-for-debugger-connectiondebugger-connecteddebugger-connection-lostshutting-down
